chapters/initial-data-splitting.qmd

```diff
@@ -362,15 +380,102 @@ When splitting multi-level data into a training and test set, the data are split
+
+See @Frescino2023 and @white2024smallareaestimationforest for similar data sets and background on the original problem. The Washington State data contains `r format(forested_n, big.mark = ",")` locations. The data are shown in @fig-forested where the points are colored by the outcome class (green representing forestation). The rate for forestation in these data is nearly even at `r round(mean(forested::forested$forested == "Yes") * 100, 1)`%.  
+
+```{r}
+#| label: fig-forested
+#| echo: false
+#| fig-cap: Area locations in Washington State sampled for determining forestation. Green points reflect locations that were determined to be forested. 
+
+forested_labs <- 
+  forested::forested %>% 
+  mutate(class_col = if_else(forested == "Yes", "#218239", "#d4ad42"))
+
+leaflet() %>%
+  addProviderTiles(providers$CartoDB.PositronNoLabels) %>%
+  addCircles(
+    data = forested_labs,
+    lng = ~ lon,
+    lat = ~ lat,
+    color = ~ class_col,
+    fillColor = ~ class_col,
+    fill = TRUE,
+    opacity = .1,
+    fillOpacity = 1,
+    radius = 1500
+  ) %>%
+  setView(lng = -120.3945, lat = 47.33047, zoom = 7)
+```
+
+How should we split these data? A simple random sample is inadequate; we need to ensure that adjacent/nearby locations are not allocated to the training and testing sets. There are a few ways to compensate for this issue: 
+
+ - Group the data into local areas and iterate by removing one or more group at a time. If there is no rational, objective process for creating groups, use clustering methods or create a regular grid across the data range [@Roberts2017]. 
+ - Buffering [@brenning2005spatial;@le2014spatial] is a method that excludes data around a specific data point or group. 
+ 
+We'll use both of these methods below for illustration. @Pohjankukka2017, @lyons2018comparison, @meyer2019importance @karasiak2022spatial, and @mahoney2023assessing have good overviews of the importance of countering spatial autocorrelation and appropriate sampling methods. 
+
+For our example, we’ll use blocking and buffering to allocate roughly 20% of the data to the testing set. First, the locations were allocated to a 25 x 25 grid of hexagons across the state. After this, a buffer of `r format(80^2, big.mark = ",")` squared map units was created around the hexagon such that outside points within the buffer would be excluded from being used in the training or testing sets. Finally, roughly 20% of the locations were selected for the test set by moving through the space and selecting hexagons sequentially. @fig-forested-split shows the results of this process. 
```

Comment:
"Blocking" is not a word that had yet appeared in this subsection. Does this refer to the first bullet point above?

Reply:
A switch from present and future tenses to past tense happens at "First, ..." and switches back after two paragraphs.